README.md

```diff
@@ -51,3 +53,23 @@ poetry run python ./zcash-issue-dag.py
 ```
 
 
+## Troubleshooting Setup Issues
+
+### Failed to build `pygraphviz`
+
+When running `poetry install` the command fails with this error
+
+````bash
+        pygraphviz/graphviz_wrap.c:2711:10: fatal error: 'graphviz/cgraph.h' file not found
+        #include "graphviz/cgraph.h"
+                 ^~~~~~~~~~~~~~~~~~~
+        1 error generated.
+        error: command '/usr/bin/clang' failed with exit code 1
+        [end of output]
+````
+
+**Cause:** Your environment does not find `graphviz` as a transitive dependency
+**Solution:** 
+Install `graphviz` through the adecquate means for your Operating System. Example: for MacOS `brew install graphviz`
```

Comment:
This actually points to a different problem: we don't document the `graphviz` dependency. Instead of adding a troubleshooting section, we should document that the user needs to install both `poetry` and `graphviz`.

Reply:
In fact, I just realised that #10 made those changes, which I've just reviewed and merged. Please check whether the current `README` is sufficient for your needs, and/or rebase this PR to make any remaining changes you want.

Reply:
Yeah I covers it